adapter/resources/apis/test.yaml

```diff
@@ -0,0 +1,360 @@
+openapi: "3.0"
```

Comment:
What's the use of this file? Just wondering whether it needs to be move to an example sort of folder